src/lib/session.ts

```diff
@@ -36,7 +36,7 @@ const getTtlMs = (cookie: CookieSerializeOptions): number => {
 
 	if (maxAge) return maxAge * 1000;
 	if (expires) {
-		const ms = Number(expires) - Date.now();
+		const ms = new Date(expires).getTime() - Date.now();
```

Comment:
Thank you, I was omitting to take into account that the `Date `object serialized with `JSON.stringify()` is a string.